### PR TITLE
On/Offboaring updates

### PR DESCRIPTION
**Pull request by ctro**
- Clarify resignation letter for contractors
- Remove `identity-admins` for partial offboarding
- Add HRLinks task for onboarding buddy

### CODE REVIEW

File: _articles/onboarding.md [hunk 1]
```diff
@@ -51,6 +51,7 @@ Look at this work as a power multiplier, you are helping someone gain a firm fou
   - Sprint ceremonies
   - All-hands / Demo-day
   - Retros / IRLs
+- Help your buddy double check that they are on the correct Performance Plan in HRLinks
```
Comment (zachmargolis) [inline _articles/onboarding.md:54]: How do you know they're on the right or wrong plan?

File: _articles/offboarding.md [hunk 2]
```diff
@@ -14,8 +14,8 @@ Review the [Leaving TTS page in the TTS Handbook](https://handbook.tts.gsa.gov/l
 
 ### For offboarding assistant to complete
 
-- Review and share [leaving GSA/TTS guidance](https://handbook.tts.gsa.gov/leaving-tts/) with this person.
-  - If this person is unable to [email their resignation letter](https://handbook.tts.gsa.gov/leaving-tts/#1-email-your-resignation-letter) for any reason you must do it on their behalf.
+- If this person is leaving GSA/TTS, review and share [leaving GSA/TTS guidance](https://handbook.tts.gsa.gov/leaving-tts/).
+  - If this person is unable to [email their resignation letter](https://handbook.tts.gsa.gov/leaving-tts/#1-email-your-resignation-letter) for any reason you must do it on their behalf. **This applies to contractors**.
```
Comment (zachmargolis) [inline _articles/offboarding.md:18]: Can we clarify this say whether it applies to "contractors, too" or "just contractors"

**Review by zachmargolis — APPROVED** [review]
LGTM, some small comments

File: _articles/onboarding.md (hunk 1 above)
Comment (ctro) [reply _articles/onboarding.md]: Good Q.  It's mostly about making sure they can get into HRLinks and have a Performance Plan, made an update.

File: _articles/offboarding.md (hunk 2 above)
Comment (ctro) [reply _articles/offboarding.md]: Yes, thanks, updated.

**Pull request merged**